x-pack/filebeat/module/panw/panos/test/pan_inc_threat.log-expected.json

```diff
@@ -5065,7 +5324,9 @@
             "pan-os",
             "forwarded"
         ],
-        "url.original": "uLLGRaXP.exe"
+        "url.domain": "uLLGRaXP.exe",
+        "url.original": "uLLGRaXP.exe",
```

Comment:
This looks like a filename not a URL.  It looks like we need to check the sub_type to see if this field is a url or file.  https://docs.paloaltonetworks.com/pan-os/8-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields.html. This would be a bigger change, but what do you think about adding an earlier processor to figure out if this is really a file or url?